### DISCUSSION TITLE
Any plan of releasing 1.5.6?

### DISCUSSION THREAD

**Discussion by egolearner** [bytedeco/javacpp#507] [General]
Hi @saudet , do you have any schedule regarding with releasing 1.5.6? 
We depend on some parser improvements of 1.5.6.

**Comment by saudet** [comment]
Most likely sooner rather than later. Within a month maybe?

**Reply by egolearner** [reply]
Thanks and looking forward!

**Reply by saudet** [reply]
JavaCPP 1.5.6 has been released. Enjoy!
